docs/designs/rubygems-001.md

```diff
@@ -0,0 +1,547 @@
+# Initial Design for the Rubygems Package Manager
+
+- [Overview](#overview)
+- [Context](#context)
+- [Description](#description)
+    - [Ruby ecosystem overview](#i-ruby-ecosystem-overview)
+    - [Current implementation overview (Cachito)](#ii-overview-of-the-current-implementation-in-cachito)
+    - [Design for the Cachi2 implementation](#iii-design-for-the-implementation-in-cachi2)
+- [Decision](#decision)
+
+## Overview
+
+Design that covers the initial implementation of the Rubygem/Bundler package manager in Cachi2. It takes inspiration from the original implementation done in [Cachito](https://github.com/containerbuildsystem/cachito).
+
+| | |
+|---------|---------|
+|Author |[Michal Šoltis](msoltis@redhat.com) |
+|Co-author |[Bruno Pimentel](bpimente@redhat.com) |
```

Comment:
I think a single `Authors` field is enough and then list all participants in an order respecting their amount of contribution, credit, etc.

Reply:
We dropped the authors altogether, resolving.